demo/src/layout/Main.js

```diff
@@ -112,9 +128,19 @@ class Main extends Component {
             <Route path="/icon" render={() => <IconDoc />} />
             <Route path="/signup" component={SignupForm} />
             <Route path="/query-form" component={QueryForm} />
-            <Route path="/theme" component={Theme} />
+            <Route path="/full-screen-loader" component={FullScreenLoader} />
```

Comment:
this goes back to the `fullscreen-loader` or `full-screen-loader` argument. If we are doing `Fullscreen` vs `Full Screen` then the route should probably be `fullscreen-loader`